src/phoenix/trace/fixtures.py

```diff
@@ -200,13 +200,16 @@ def get_trace_fixture_by_name(fixture_name: str) -> TracesFixture:
 def download_traces_fixture(
     fixture: TracesFixture,
     host: Optional[str] = "https://storage.googleapis.com/",
-    bucket: Optional[str] = "arize-assets",
-    prefix: Optional[str] = "phoenix/traces/",
+    bucket: Optional[str] = "arize-phoenix-assets",
+    prefix: Optional[str] = "traces/",
 ) -> List[str]:
     """
     Downloads the traces fixture from the phoenix bucket.
     """
     url = f"{host}{bucket}/{prefix}{fixture.file_name}"
+    if is_parquet_file(url):
+        return [url]
+
```

Comment:
`download_traces_fixture` is not downloading the data for Parquet files, the download is happening in `load_example_traces`. Can we rearrange things so the name of the function more accurately reflects where the download is happening? Both functions may no longer be necessary.